packages/next/src/server/lib/router-server.ts

```diff
@@ -428,11 +428,11 @@ export async function initialize(opts: {
             fsChecker.pageFiles.has(matchedOutput.itemPath))
         ) {
           res.statusCode = 500
+          const message = `A conflicting public file and page file was found for path ${matchedOutput.itemPath} https://nextjs.org/docs/messages/conflicting-public-file-page`
+          console.error(message)
```

Comment:
Should use nextjs logger, and we have a test for this error. Need to update the tests for checking next CLi output as well